python/lsst/ts/salobj/sal_info.py

```diff
@@ -766,12 +765,17 @@ async def _read_loop(self, loop: asyncio.AbstractEventLoop) -> None:
 
     def _blocking_read(
         self,
-    ) -> typing.Tuple[
-        typing.Optional[topics.ReadTopic], typing.List[type_hints.BaseMsgType]
+    ) -> typing.Optional[
```

Comment:
Please remove typing.Optional and return an empty list if nothing is read.

Also please consider changing `typing.List` to `list` and `typing.Tuple` to `tuple`. I plan to take a global pass on salobj for this, but meanwhile you may as well update new code?

Reply:
Removed Optimal